### PR TITLE
Add instructions for linux dependencies

### PR DESCRIPTION
**Pull request by meowsbits**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,12 +39,41 @@ If you are on OSX you can get them via [brew](https://github.com/Homebrew/brew)
...
+LatestVirtualBoxVersion=$(wget -qO - http://download.virtualbox.org/virtualbox/LATEST-STABLE.TXT) && wget "http://download.virtualbox.org/virtualbox/${LatestVirtualBoxVersion}/Oracle_VM_VirtualBox_Extension_Pack-${LatestVirtualBoxVersion}.vbox-extpack"
+
+sudo VBoxManage extpack install --replace Oracle_VM_VirtualBox_Extension_Pack-${LatestVirtualBoxVersion}.vbox-extpack
+```
+> https://unix.stackexchange.com/questions/289685/how-to-install-virtualbox-extension-pack-to-virtualbox-latest-version-on-linux/290462
+
 # Usage
 
-Add the following entry to your `/etc/hosts` file:
+Add the `10.10.10.100 jenkins.example.com` to your `/etc/hosts` file.
```
Comment (BelfordZ) [inline README.md]:
```suggestion
Add `10.10.10.100 jenkins.example.com` to your `/etc/hosts` file. 
```

**Comment by shanejonas** [comment]
This is awesome.

Thank you for taking the time to make this contribution.

thank-you

**Comment by meowsbits** [comment]
@spllchckr @BelfordZ were iz spilling eror?

**Comment by shanejonas** [comment]
LGTM

image

**Comment by meowsbits** [comment]
image

**Comment by shanejonas** [comment]
@meowsbits you need to sign your commits and fix the messages to align with conventional commits

File: README.md [hunk 2]
```diff
@@ -39,12 +39,41 @@ If you are on OSX you can get them via [brew](https://github.com/Homebrew/brew)
...
 Installing virtualbox requires you to allow `Oracle America, Inc` in `System Preferences > Security & Privacy` in order to finish the installation. More information can be found [here](http://osxdaily.com/2018/12/31/install-run-virtualbox-macos-install-kernel-fails/).
 
+
+## Ubuntu Installation Instructions:
+
+Virtualbox:
+
+```
+sudo apt-add-repository "deb http://download.virtualbox.org/virtualbox/debian $(lsb_release -sc) contrib"
+`wget -q https://www.virtualbox.org/download/oracle_vbox.asc -O- | sudo apt-key add -
```
Comment (shanejonas) [inline README.md]:
```suggestion
wget -q https://www.virtualbox.org/download/oracle_vbox.asc -O- | sudo apt-key add -
```

**Comment by meowsbits** [comment]
As for signing commits, the important thing is that merge commits are signed, right?

**Comment by BelfordZ** [comment]
@meowsbits preferable that all commits are signed.

**Comment by BelfordZ** [comment]
@meowsbits https://help.github.com/articles/managing-commit-signature-verification/

**Comment by meowsbits** [comment]
Just a heads up that this can cause/enable/encourage some strange-ish/awkward things. 

For example the second-to-last commit on this branch, now 1e84eb7. 

```
commit 1e84eb77ee7d6544588d46a7fd41416690a5a943 (HEAD -> feat/linux-install-instructions, origin/feat/linux-install-instructions)
Author: Shane <jonas.shane@gmail.com>
Date:   Fri Feb 8 17:44:52 2019 -0600

    Update README.md
    
    Co-Authored-By: leroyjenkins <4560033b+meowsdbits@users.noreply.github.com>
    
    But-Really-It-Was: Shane Jonas, MD

diff --git a/README.md b/README.md
index 663d0e5..463f562 100644
--- a/README.md
+++ b/README.md
@@ -46,7 +46,7 @@ Virtualbox:

```
screenshot_20190208_213658

It requires a force push (or 14) because the commit was made on Github.

(Not to mention instead of using `--no-edit` I could easily make any change to the patch, and it would look like Shane authored it, which seems like it might have negative human side effects) 

Or just don't use/allow Github on-PR `suggestion`s?

**Comment by shanejonas** [comment]
please use one of the conventional commit suggestions of `fix` or `feat` so we can merge this

**Pull request merged**